spec/controllers/webhook_events_controller_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe WebhookEventsController, type: :controller do
+  include ActiveJob::TestHelper
+
+  let(:organization)  { GitHubFactory.create_owner_classroom_org }
+  let(:user)          { organization.users.first                 }
+
+  describe 'POST #create' do
+    describe 'ping event' do
+      context 'valid payload signature' do
+        before(:each) do
+          params = {
+            zen: 'Design for failure.',
+            hook_id: 999,
+            hook: {
+              type: 'Organization',
+              id: 999
+            },
+            sender: {
+              id: user.uid
+            },
+            organization: {
+              id: organization.github_id
+            }
+          }
```

Comment:
Good idea! However, we'll need to create the `organization` and `sender` manually since we need them to be the same as the ones in the stored JSON payload.